dotnet/simple-app/Program.cs

```diff
@@ -0,0 +1,194 @@
+﻿/*
+Copyright 2022 CodeNotary, Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+using System.Text;
+using ImmuDB;
+using ImmuDB.Exceptions;
+
+namespace simple_app;
+
+class Program
+{
+    public static async Task Main(string[] args)
+    {
+        await InitializeExample();
+        await AnotherInitializeExample();
+        await YetAnotherInitializeExample();
+        await MiscFunctionsUsageExample();
+        await ImmuClient.ReleaseSdkResources();
+    }
+
+    private static async Task InitializeExample()

```

Comment:
initialize what ?

Reply:
what is the difference between those 3 examples ?

Reply:
See the source code, the way `client` object is constructed. I will rename it to OpenConnection.

Reply:
Added Readme.md where the functions in the code sample are described.